We do not have a 128-bit long double implementation for PA, so do
not imply that we need the 128-bit long double support in glibc.
We will likely define long double equivalent to double.

2005-06-18  Carlos O'Donell <carlos@systemhalted.org>

	* sysdeps/hppa/hppa1.1/Implies: Remove ldbl-128.

===================================================================
RCS file: /var/lib/cvs/glibc/sysdeps/hppa/hppa1.1/Implies,v
retrieving revision 1.2
retrieving revision 1.3
diff -u -r1.2 -r1.3
--- glibc/sysdeps/hppa/hppa1.1/Implies	2005/06/08 20:32:10	1.2
+++ glibc/sysdeps/hppa/hppa1.1/Implies	2005/07/18 04:32:31	1.3
@@ -1,4 +1,3 @@
 wordsize-32
 ieee754/flt-32
 ieee754/dbl-64
-ieee754/ldbl-128
